packages/extension-base/src/page/PostMessageProvider.ts

```diff
@@ -74,15 +73,13 @@ export default class PostMessageProvider implements InjectedProvider {
    */
   // eslint-disable-next-line @typescript-eslint/require-await
   public async disconnect (): Promise<void> {
-    // FIXME This should see if the extension's state's provider can disconnect
     console.error('PostMessageProvider.disconnect() is not implemented.');
   }
 
   /**
    * @summary `true` when this provider supports subscriptions
    */
   public get hasSubscriptions (): boolean {
-    // FIXME This should see if the extension's state's provider has subscriptions
```

Comment:
I don't think we are interested in implementing this feature

Reply:
Not sure if I understand the _state's provider_ concepts, but I think that we only have this: `packages/extension-base/src/background/handlers/State.ts` state provider, which being static makes it reasonable to have `true` hardcoded in here. If my reasoning sounds sane, I'd put a comment explaining that:
```suggestion
// Our state provider (packages/extension-base/src/background/handlers/State.ts) supports subscriptions
```